### PR TITLE
Add AWS CLI capabilities

### PR DESCRIPTION
**Pull request by tschleining**

### CODE REVIEW

File: src/Misc/layoutbin/powershell/Add-AwsCliCapabilities.ps1 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[CmdletBinding()]
+param()
+
+Add-CapabilityFromApplication -Name 'AWS_Cli' -ApplicationName 'aws'
```
Comment (TingluoHuang) [inline src/Misc/layoutbin/powershell/Add-AwsCliCapabilities.ps1]: Should we do a xplat capabilities, since awscli is xplat?

Comment (tschleining) [inline src/Misc/layoutbin/powershell/Add-AwsCliCapabilities.ps1]: Great suggestion!  I didn't even think of that.

Comment (tschleining) [inline src/Misc/layoutbin/powershell/Add-AwsCliCapabilities.ps1]: Not sure where I should start with this.  Can you point me in the right direction?

Comment (TingluoHuang) [inline src/Misc/layoutbin/powershell/Add-AwsCliCapabilities.ps1]: You can take reference of npm, gulp, and other node tool capability. basic you will have the .ps1 for windows, and modify /src/Agent.Listener/Capabilities/NixCapabilitiesProvider.cs for linux and osx.  src/Misc/layoutbin/powershell/Add-NodeToolCapabilities.ps1

**Pull request closed without merging**